PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-storage/src/main/java/com/microsoft/azure/toolkit/intellij/storage/connection/BaseStorageAccountResourceDefinition.java

```diff
@@ -94,4 +96,22 @@ public List<String> getEnvironmentVariablesKey() {
     public String getResourceConnectionString(@Nonnull IStorageAccount resource) {
         return resource instanceof AzuriteStorageAccount ? LOCAL_STORAGE_CONNECTION_STRING : resource.getConnectionString();
     }
+
+    @Override
+    public Map<String, String> initIdentityEnv(AzureServiceResource<IStorageAccount> accountDef, Project project) {
+        final HashMap<String, String> env = new HashMap<>();
+        final IStorageAccount account = accountDef.getData();
+        if (Objects.nonNull(account)) {
+            env.put(ACCOUNT_NAME_KEY, account.getName());
+        }
+        return env;
+    }
+
+    @Nullable
+    @Override
+    public Map<String, BuiltInRole> getBuiltInRoles() {
+        return Map.of("ba92f5b4-2d11-453d-a403-e96b0029c9fe", BuiltInRole.STORAGE_BLOB_DATA_CONTRIBUTOR,
```

Comment:
add comments about these ids?

Reply:
Added the references in `IManagedIdentitySupported`